apps/fz_http/test/support/api_case.ex

```diff
@@ -0,0 +1,61 @@
+defmodule FzHttpWeb.ApiCase do
+  @moduledoc """
+  This module defines the test case to be used by
+  tests that require setting up a connection.
+
+  Such tests rely on `Phoenix.ConnTest` and also
+  import other functionality to make it easier
+  to build common data structures and query the data layer.
+
+  Finally, if the test case interacts with the database,
+  we enable the SQL sandbox, so changes done to the database
+  are reverted at the end of every test. If you are using
+  PostgreSQL, you can even run database tests asynchronously
+  by setting `use FzHttpWeb.ConnCase, async: true`, although
+  this option is not recommended for other databases.
+  """
+  use ExUnit.CaseTemplate
+  use FzHttp.CaseTemplate
+
+  using do
+    quote do
+      # Import conveniences for testing with connections
+      import Plug.Conn
+      import Phoenix.ConnTest
+      import FzHttp.TestHelpers
+
+      alias FzHttp.{
+        ApiTokensFixtures,
+        UsersFixtures
+      }
+
+      use FzHttpWeb, :verified_routes
+
+      # The default endpoint for testing
+      @endpoint FzHttpWeb.Endpoint
+
+      def new_conn do
```

Comment:
It's better to avoid adding new functions to the `using/1` macro because they will be compiled into every single module. Instead, you can add `import FzHttpWeb.ApiCase` and define functions inside the module itself.

Reply:
Fixed